src/test/java/io/numaproj/numaflow/function/FunctionServerTest.java

```diff
@@ -152,4 +152,45 @@ public void reducer() throws InterruptedException {
 
     }
 
+    @Test
+    public void reducerWithMultipleKey() {
+        String reduceKey = "reduce-key";
+        int keyCount = 100;
```

Comment:
Can we use a smaller key count? I am not sure if it's gonna help make unit test run faster but functionally speaking, `100` and `2` both fulfill the purpose of testing` reducerWithMultipleKey` right?

Reply:
100 is a small number..